synapse/lib/grammar.py

```diff
@@ -3,7 +3,7 @@
 import synapse.exc as s_exc
 
 # TODO:  commonize with storm.lark
-re_scmd = '[a-z][a-z0-9.]+'
+re_scmd = '^[a-z][a-z0-9.]+$'
```

Comment:
Will this catch data set via the ``CoreApi.setStormCmd()`` endpoint?

Reply:
Yeah setStormCmd uses this same regex for testing command name validity.

Reply:
It was actually using it in a way where it already worked